packages/@react-spectrum/s2/src/TableView.tsx

```diff
@@ -457,7 +457,8 @@ const cellFocus = {
   outlineOffset: -2,
   outlineWidth: 2,
   outlineColor: 'focus-ring',
-  borderRadius: '[6px]'
+  borderRadius: '[6px]',
+  pointerEvents: 'none'
```

Comment:
what what the reason for the `pointerEvents: 'none'` here? I'm sure its fine, but curious what the original bug was, just a press on the focus ring itself?